qa/suites/rados/verify/tasks/rados_api_tests.yaml

```diff
@@ -1,7 +1,8 @@
 teuthology:
   postmerge:
-    # Don't run this test on tiny clusters
-    - if yaml.clusternodes < 4 then reject() end
+    - |
+      -- Don't run this test on tiny clusters
+      if py_len(yaml.roles) < 4 then reject() end
```

Comment:
@badone  Unless I've confused myself, I don't think `rados:verify` is ever run with more than 2 nodes so I don't think this is ever tested. OTOH, `rados:thrash` always has 4 nodes so the postmerge always succeeds.

Am I missing something?

Reply:
https://pulpito.ceph.com/yuriw-2024-05-02_23:59:28-rados-wip-yuriw11-testing-20240501.200505-squid-distro-default-smithi/7686880/ is an example of the combination on a fixed-2 cluster that I'm trying to avoid.

Reply:
I see, in `squid` the `rados:thrash` suite is using `qa/clusters/fixed-2.yaml`.

This fix is appropriate even if the cluster is updated to 4 nodes in `main` and beyond. IMO, a test (or in this case, a fragment for a test) should self-document its requirements.